docs/modules/migration/partials/migration-guide-24.1.adoc

```diff
@@ -263,11 +270,14 @@ Dependencies relevant for all builds should be placed in the `.app` module.
 Make sure to move your dependencies accordingly.
 
 The Scout archetypes (Classic and JS) were updated and generate the new `.app` module structure.
-It's highly advised to update your project structure as newly defined.
+Update your project structure as newly defined.
 This includes the migration from `web.xml` to servlet contributors and the removal of the `.war` module(s) as described in the next chapters too.
 
+[[migration-guide-24-1-jetty-web.xml]]
 === web.xml
 
+Using a `web.xml` to define servlet filters and servlet is no longer supported.
```

Comment:
mention that web.xml support was dropped if using the deployment with embedded jetty server (preferred deployment option).
web.xml support is still available if using a web app container deployment using a war.

Reply:
Migration guide improved accordingly